app/controllers/api/invitations_controller.rb

```diff
@@ -10,6 +10,18 @@ def create
     respond_with_collection
   end
 
+  def pending
+    load_and_authorize :group
+    @invitations = page_collection(@group.invitations.pending)
+    respond_with_collection
+  end
+
+  def destroy
+    @invitation = Invitation.find(params[:id])
```

Comment:
This should be done in the InvitationService, where it will check that you have permission to do this.